### ISSUE TITLE
Sonarr retries download a torrent (and fails) if it exists in qBitTorrent with a different name

### ISSUE THREAD

**Issue by xddxdd** [Sonarr/Sonarr#5336]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

I'm using Sonarr to download an anime episode. Sonarr correctly matched the episode and downloaded it with qBitTorrent.

However, the task in qBitTorrent exists with a different name from the name shown on the tracker site.

If I perform search on the same episode again, Sonarr will retry downloading the same torrent, and because it already exists in qBitTorrent, qBitTorrent returns an error. Sonarr will then retry and download a different source from a different group, which I do not want.

### Expected Behavior

Sonarr should detect that the task for this episode still exists in qBitTorrent, and do not try to download it again.

Alternatively, Sonarr can force set the task name in qBitTorrent when it's created.

### Steps To Reproduce

The tracker page for the torrent is: <https://share.dmhy.org/topics/view/616829_LoliHouse_Bocchi_the_Rock%21_-_01_WebRip_1080p_HEVC-10bit_AAC.html>

The direct link to the torrent is: <https://dl.dmhy.org/2022/10/13/bf82b074a6f1d8ea06a1425c0c335ae1490d3a05.torrent>

The name on the tracker page is: `[动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 01 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]`

The task name in qBitTorrent is: `[DMG&LoliHouse] Bocchi the Rock! - 01 [WebRip 1080p HEVC-10bit AAC ASSx2].mkv`

I'm using the `dmhy` indexer from Prowlarr.

### Environment

```markdown
- OS: NixOS 22.11
- Sonarr: 3.0.9.1549
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Firefox 108.0.1
```

### What branch are you running?

Main

### Trace Logs?

This is not a trace log, but it already shows the failure to add torrent to qBitTorrent, the retries with the same torrent from different trackers, and finally success with a different torrent from another group.

Let me know if you need a reproduce with a trace log.

```
Jan 02 23:48:13 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 01 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:13 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:13 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 01 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:15 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:15 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / Bocchi the Rock! - 01 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:18 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromFile (System.String fileName, System.Byte[] fileContent, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x0005f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromTorrentFile (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String filename, System.Byte[] fileContent) [0x000ae] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x00230] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:18 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / Bocchi the Rock! - 01 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:20 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:20 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 02 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:21 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:21 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 02 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:23 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:23 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 02 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:26 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromFile (System.String fileName, System.Byte[] fileContent, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x0005f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromTorrentFile (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String filename, System.Byte[] fileContent) [0x000ae] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x00230] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:26 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]: [Warn] ProcessDownloadDecisions: Couldn't add report to download queue. [动漫国字幕组&LoliHouse] 孤独摇滚 / BOCCHI THE ROCK! - 02 [WebRip 1080p HEVC-10bit AAC][简繁内封字幕]
Jan 02 23:48:28 NzbDrone[361676]: [v3.0.9.1549] NzbDrone.Core.Download.Clients.DownloadClientException: Download client failed to add torrent by url
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentProxyV2.AddTorrentFromUrl (System.String torrentUrl, NzbDrone.Core.Download.Clients.TorrentSeedConfiguration seedConfiguration, NzbDrone.Core.Download.Clients.QBittorrent.QBittorrentSettings settings) [0x00056] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.Clients.QBittorrent.QBittorrent.AddFromMagnetLink (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String hash, System.String magnetLink) [0x000de] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromMagnetUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String magnetUrl) [0x00048] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].DownloadFromWebUrl (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode, System.String torrentUrl) [0x000bc] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.TorrentClientBase`1[TSettings].Download (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x00148] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.DownloadService.DownloadReport (NzbDrone.Core.Parser.Model.RemoteEpisode remoteEpisode) [0x0019f] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:28 NzbDrone[361676]:   at NzbDrone.Core.Download.ProcessDownloadDecisions.ProcessDecisions (System.Collections.Generic.List`1[T] decisions) [0x00104] in <0b43b6e47ef8497b83f58230dd1ec559>:0
Jan 02 23:48:29 NzbDrone[361676]: [Info] DownloadService: Report sent to qBitTorrent. 【爪爪字幕组】★10月新番[孤独摇滚！/Bocchi the Rock!][01][1080p][HEVC][简日双语][MP4][招募翻译校对]
```

### Anything else?

This is a screenshot of the history section of the episode:

<img width="942" alt="Screenshot_20230103_004330" src="https://user-images.githubusercontent.com/5778879/210311144-9effd818-305a-4b1e-bd07-9bd34ae89c7b.png">

qBitTorrent returns error for existing torrents in this section of their source code:

https://github.com/qbittorrent/qBittorrent/blob/c5a4a0db2cae5bfcb81cbc395f47fca49ef36aff/src/base/bittorrent/sessionimpl.cpp#L2644-L2658

**Comment by bakerboy448** [comment]
Qbit does not give any useful error response when a duplicate is inserted and simply gives a generic "error"  there's not much sonarr can really do here.

**Comment by xddxdd** [comment]
Is it possible to add an option in Sonarr to ignore errors from qBit?

From reading qBit source code, it should only return error when the torrent is corrupted, or the torrent exists. And IMO it would be really rare for torrents downloaded from trackers to be corrupted.

**Comment by markus101** [comment]
Qbit returns a successful response (200 OK) and the content of `Fails.` for duplicate torrents. If they can provide a meaningful message in response to adding a duplicate torrent then we could work with that, but as it stands right now with Sonarr having no way to differentiate between a failure and a "failure" it's going to fail and try another release.

We're not going to add an option to ignore errors.

Sonarr should be aware of the download in the queue already and reject it for already being there, unless for some reason it can't parse/match it to the series and episode.
With `Show Unknown Series Items` enabled for Activity: Queue

![image](https://user-images.githubusercontent.com/413544/210446779-01bd30f4-0cb4-48c8-b3b6-e45964579a67.png)

What shows under Activity: Queue for that download?

**Comment by bakerboy448** [comment]
Follow up to markus's question - is the existing torrent in sonarr's category?

**Comment by xddxdd** [comment]
I guess I know what's happening. I have a release profile that matches the title in the tracker, but not the title in qBitTorrent. Sonarr thinks the release on the tracker is better, although they are actually the same release.

This is the activity history for the download request:

<img width="1092" alt="Screenshot_20230103_162952" src="https://user-images.githubusercontent.com/5778879/210452173-76a032c5-804d-4821-9c89-a0d9c50fe43e.png">

And the activity history when the download finished:

<img width="1092" alt="Screenshot_20230103_162909" src="https://user-images.githubusercontent.com/5778879/210452186-9c0b3309-2b16-4568-82f3-18a80424ebb1.png">

The torrent is in the tv-sonarr category in qBitTorrent.

For now I will patch qBitTorrent myself to stop reporting duplicate torrents as errors.

**Comment by markus101** [comment]
Closing this as Sonarr doesn't know why qbit failed the release and isn't able to determine if that same release was added under a different name.

**Comment by bakerboy448** [comment]
Opened https://github.com/qbittorrent/qBittorrent/issues/18361
